### PR TITLE
feat: added Lgamma to paddle frontend

### LINKED ISSUE #26351: lgamma
add lgamma to Tensor Instance Methods to Paddle Frontend

### PR DESCRIPTION
**Pull request by lukaM19**
# PR Description
added lgamma frontend to paddle

## Related Issue

Close #26351 

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by lukaM19** [comment]
I don't think my PR introduces any new errors, let me know if anything else needs to be changed !

**Comment by lukaM19** [comment]
I added the function to the tensor class, I still think no new failed tests are introduced by my code, let me know if I should edit something

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -106,6 +106,10 @@ def dim(self):
     def abs(self):
         return paddle_frontend.abs(self)
 
+    @with_unsupported_dtypes({"2.5.1 and below": ("float16",)}, "paddle")
+    def lgamma(self):
+        return ivy.lgamma(self)
```
Comment (YushaArif99) [inline ivy/functional/frontends/paddle/tensor/tensor.py:111]: this should call `paddle_frontend.lgamma`

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_manipulation.py
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_manipulation.py:1]: no need to add a separate test for `paddle.lgamma`. So you can remove these changes 🙂

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @lukaM19. The changes look good to me at a first glance. There are a few small changes I have requested above. 
I am also noticing the CI tests for `test_paddle_tensor_lgamma` failing [here](https://github.com/unifyai/ivy/actions/runs/6370450261/job/17291572370?pr=26376#step:3:198) with the following error: 

```python
ret = torch.as_tensor(obj, dtype=dtype, device=device)
E   TypeError: object of type 'Tensor' has no len()
```

Could you kindly look into this and ensure the relevant CI tests are all passing? Feel free to request a review once this is done. Thanks! 🙂

**Comment by lukaM19** [comment]
@YushaArif99  I made some changes and none of the tests associated with my changes and lgamma seem to fail, I would love to know if you think I should change something else

**Comment by lukaM19** [comment]
@YushaArif99 just wondering if you could take another look at the PR

**Pull request closed without merging**